src/tooltip.jsx

```diff
@@ -57,6 +57,7 @@ let Tooltip = React.createClass({
         borderRadius: 2,
         userSelect: 'none',
         opacity: 0,
+        zIndex: 99,
```

Comment:
We should hold off on this change - please see #719. I think we need to come up with a zindex strategy for all components. In the mean time, you should be able to fix the tooltip is above other things by making sure the IconButton has the proper z-index.